layers/README.md

```diff
@@ -0,0 +1,118 @@
+
+| method [x = Layer()]| description |
+|:-------|:------------|
+| x.load_input(_input_) | Loads an ATT&CK layer from either a dictionary or a string representation of a dictionary. |
+| x.load_file(_input_) | Loads an ATT&CK layer from a file location specified by the _input_. |
+| x.export_file(_input_) | Saves the current state of the loaded ATT&CK layer to a json file denoted by the _input_. |
+| x.get_dict() | Returns a representation of the current ATT&CK layer object as a dictionary. | 
+
+#### Example Usage
+
+```python
+example_layer_dict = {
+    "name": "example layer",
+    "version": "3.0",
+    "domain": "mitre-enterprise"
+}
+
+example_layer_location = "/path/to/layer/file.json"
+example_layer_out_location = "/path/to/new/layer/file.json"
+
+from layers.core import Layer
+
+layer1 = Layer(example_layer_dict)              # Create a new layer and load existing data
+layer1.export_file(example_layer_out_location)  # Write out the loaded layer to the specified file
+
+layer2 = Layer()                                # Create a new layer object
+layer2.load_input(example_layer_dict)           # Load layer data into existing layer object
+print(layer2.get_dict())                        # Retrieve the loaded layer's data as a dictionary, and print it
+
+layer3 = Layer()                                # Create a new layer object
+layer3.load_file(example_layer_location)        # Load layer data from a file into existing layer object
+```
+
+## layerops.py
+Layerops.py provides the LayerOps class, which is a way to combine layer files in an automated way, using user defined lambda functions. Each LayerOps instance, when created, ingests the provided lambda functions, and stores them for use. An existing LayerOps class can be used to combine layer files according to the initialized lambda using the process method. The breakdown of this two step process is documented in the table below, while examples of both the list and dictionary modes of operation can be found below.
+
+##### LayerOps()
+```python
+ x = LayerOps(score=score, comment=comment, enabled=enabled, colors=colors, metadata=metadata, name=name, desc=desc, default_values=default_values)
+```
+ 
+ Each of the _inputs_ takes a lambda function that will be used to combine technique object fields matching the parameter. The one exception to this is _default_values_, which is an optional dictionary argument containing default values to provide the lambda functions if techniques of the combined layers are missing them.
+
+##### .process() Method
+```python
+x.process(data, defaults=defaults)
+```
+The process method applies the lambda functions stored during initialization to the layer objects in _data_. _data_ must be either a list or a dictionary of Layer objects, and is expected to match the format of the lambda equations provided during initialization.
+
+#### Example Usage
+```python
+from layers.manipulators.layerops import LayerOps
+from layers.core.layer import Layer
```

Comment:
This doesn't seem to work for me:

```
$ python3
Python 3.7.4 (default, Sep  7 2019, 18:27:02)
[Clang 10.0.1 (clang-1001.0.46.4)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from layers.core.layer import Layer
>>> from layers.manipulators.layerops import LayerOps
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/isabel/working/temp/attack-scripts/layers/manipulators/layerops.py", line 23, in <module>
    from layers.core import Layer
ImportError: cannot import name 'Layer' from 'layers.core' (unknown location)
>>>
```
Changing the import at the beginning of the layerops script to `from layers.core.layer import Layer` seems to fix the issue, but it assumes the working directory is the root of this repo. If I'm running from the `layers` folder and run `from manipulators.layerops import LayerOps`, the absolute import breaks. 

I'm not sure what the best practice is with this — a relative import, or just tell the user to always import from the repo root folder? If the latter, please specify where this is being run from in the example as well as putting towards the top of the file "all scripts should be imported and run from the root folder of this repository."

Long term if this is going to become a pip package this probably needs to be relative. Not sure if it's worth doing now.

Reply:
Speaking of importing, this conversation doesn't seem to have been addressed: https://github.com/mitre-attack/attack-scripts/pull/15#discussion_r414033610

Reply:
It was, I just failed to notice that the updated __init__.py wasn't included in the update for some reason. I've fixed this, and it appears to work properly from wherever now.